pkg/steps/release/promote_test.go

```diff
@@ -571,18 +571,20 @@ func TestGetPromotionPod(t *testing.T) {
 			namespace: "ci-op-zyvwvffx",
 		},
 		{
-			name: "tag_by_commit is true",
```

Comment:
The bug (fixed by https://github.com/openshift/ci-tools/pull/3481/files) was hit when the same image (the same digest) is promoted to different tags (thus, it would lead to the same tag in quay.io because the tag in quay is determined by the original image's digest).
`tag_by_commit` is only its special case.

With the implementation of this PR, the problem is fixed implicitly because it just does the assignment of the map with the same key/value twice (i.e., no-op for the 2nd time) which is used to define the `oc image mirror` command.